airbyte-integrations/connectors/source-stripe/unit_tests/resource/http/response/500.json

```diff
@@ -0,0 +1,3 @@
+{
+  "unknown": "maxi297: I could not reproduce the issue hence this response will not look like the actual 500 status response"
```

Comment:
Just curious if you checked Sentry or logs for 500s. (I didn't see any with a cursory check.)

I did find this [Stripe documentation](https://stripe.com/docs/api/errors) that gives a suggestion for what fields there might be. Maybe worth updating this to look a little more like that?

Reply:
I did check Sentry and didn't find anything. As we don't have a clear example, I would prefer to fail loudly with something that is clearly not a response from the stripe API and the person that relies on the 500 HTTP response can update this

Reply:
Makes sense to me!